src/main/kotlin/lotto/domain/Ticket.kt

```diff
@@ -1,12 +1,24 @@
 package lotto.domain
 
+import lotto.controller.WinningNumbers
+
 class Ticket(val lottoNumber: Set<LottoNumber>) {
     constructor() : this(generateLottoNumber())
 
+    init {
+        require(lottoNumber.size == 6) {
+            throw IllegalArgumentException("Please enter $AMOUNT_OF_NUMBER_FOR_TICKET numbers")
+        }
+    }
+
     constructor(input: List<Int>) : this(input.map { LottoNumber(it) }.toSet())
```

Comment:
Before merging, it might be good to double-check if the class layout follows the recommended order across the code.